recipe/gen_patch_json.py

```diff
@@ -795,8 +795,27 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
         if record_name == "airflow-with-async":
             _rename_dependency(fn, record, "evenlet", "eventlet")
 
+        # Current implementation in iris allows for frequent SegFaults
+        # when netCDF4>1.6.0; iris devs are working on a solution and
+        # discussing it in https://github.com/SciTools/iris/issues/5016
+        # but a wide repodata patch is needed before a permanent solution
```

Comment:
@trexfeathers maybe this message should be changed to "We darn well fixed it - but it's still safe to keep the patch for iris <3.4.1" - up to you if you wanna change, in a simple PR

Reply:
Yeah this definitely looks like I should update it. Thanks!

Reply:
#402